### ISSUE TITLE
Explicitly check and "range" the supported Lemmy versions

### ISSUE THREAD

**Issue by ShadowJonathan** [mlemgroup/mlem#113]
While clamping the versions of lemmy should also be guided by a policy we should setup at a later time, we should also explicitly, in the app, look at the version number the server gives us, and check if we support it.

**Comment by EricBAndrews** [comment]
API functionality that's not quite raw calls and not frontend--1.1 baybee

**Comment by Sjmarf** [comment]
I don't think we do this currently? We don't have a minimum version explicitly set anywhere in the code. It's not particularly high priority though, as our minimum version is 0.18.0 and probably no-one is trying to use 0.17

**Comment by Sjmarf** [comment]
Not that important right now I don't think. No-one is using < 0.19. If we bump the minimum version higher we should consider it
